From a7631e083a56ce5948855061f95ffcccc6718683 Mon Sep 17 00:00:00 2001
From: Christophe Chapuis <chris.chapuis@gmail.com>
Date: Tue, 17 Sep 2019 18:34:18 +0000
Subject: [PATCH] Fix APIs conflicting definitions

glapi.py was already updated in upstream commit:

commit a737f865afb7290ba74cc5d7ac888dd3c0eeaf13
Author: Jose Fonseca <jfonseca@vmware.com>
Date:   Sun Nov 24 21:27:09 2019 +0000

    khronos: Update headers.
---
 specs/eglapi.py | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/specs/eglapi.py b/specs/eglapi.py
index 786ba05a..c6d3c17a 100644
--- a/specs/eglapi.py
+++ b/specs/eglapi.py
@@ -211,6 +211,9 @@ EGLPlatformDisplayAttribsEXT = EGLIntArray([
 
 EGLProc = Opaque("__eglMustCastToProperFunctionPointerType")
 
+# EGLuint64CHROMIUM
+EGLuint64CHROMIUM = Alias("EGLuint64CHROMIUM", UInt64)
+
 def GlFunction(*args, **kwargs):
     kwargs.setdefault('call', 'EGLAPIENTRY')
     return Function(*args, **kwargs)
@@ -274,7 +277,7 @@ eglapi.addFunctions([
     GlFunction(EGLBoolean, "eglQuerySurfacePointerANGLE", [(EGLDisplay, "dpy"), (EGLSurface, "surface"), (EGLint_enum, "attribute"), Out(Pointer(OpaquePointer(Void)), "value")], sideeffects=False),
 
     # EGL_CHROMIUM_get_sync_values
-    GlFunction(Bool, "eglGetSyncValuesCHROMIUM", [(EGLDisplay, "dpy"), (EGLSurface, "surface"), Out(Pointer(Int64), "ust"), Out(Pointer(Int64), "msc"), Out(Pointer(Int64), "sbc")], sideeffects=False),
+    GlFunction(EGLBoolean, "eglGetSyncValuesCHROMIUM", [(EGLDisplay, "dpy"), (EGLSurface, "surface"), Out(Pointer(EGLuint64CHROMIUM), "ust"), Out(Pointer(EGLuint64CHROMIUM), "msc"), Out(Pointer(EGLuint64CHROMIUM), "sbc")], sideeffects=False),
 
     # EGL_EXT_platform_base
     GlFunction(EGLDisplay, "eglGetPlatformDisplayEXT", [(EGLenum, "platform"), (OpaquePointer(Void), "native_display"), (EGLPlatformDisplayAttribsEXT, "attrib_list")]),
